tests/coverage/generate_coverage.sh

```diff
@@ -6,27 +6,31 @@ set -ex
 
 suffix=$1
 
+LLVM_VER=7
```

Comment:
7 or 8?

Reply:
Trying to get this from CMake, to keep it in sync/compatible.